core/shared/src/main/scala/zio/stm/ZSTM.scala

```diff
@@ -1034,6 +1034,13 @@ object ZSTM {
   def collectAll_[R, E, A](in: Chunk[ZSTM[R, E, A]]): ZSTM[R, E, Unit] =
     foreach_(in)(ZIO.identityFn)
 
+  /**
+   * Similar to Either.cond, evaluate the predicate,
+   * return the given A as success if predicate returns true, and the given E as error otherwise
+   */
+  def cond[R, E, A](predicate: Boolean, result: => A, error: => E): ZSTM[R, E, A] =
```

Comment:
Don't need `R` parameter. Can also add alias to `STM` object.

Reply:
Should I add one to TaskSTM, similar to RIO?